### ISSUE TITLE
pip3 install does not get the correct numpy

### ISSUE THREAD

**Issue by mennodeij** [precice/config-visualizer#4]
During install I get this:

`Requirement already satisfied: numpy>=1.13.3 in /usr/lib64/python3.6/site-packages (from seaborn->precice-config-visualizer==0.1.0)`

Then afterwards it complains about this
`>precice-config-visualizer --help`

`pkg_resources.ContextualVersionConflict: (numpy 1.14.3 (/usr/lib64/python3.6/site-packages), Requirement.parse('numpy>=1.14.5'), {'scipy'})`

**Comment by fsimonis** [comment]
Thank you for filing this bug report.

Can you reproduce this error in a virtual environment?

```
cd /some/folder
python -m venv myenv
. myenv/bin/activate
pip3 install https://github.com/precice/config-visualizer/archive/master.zip
precice-config-visualizer -h
```

**Comment by mennodeij** [comment]
In a virtual environment everything is OK. It looks like it is because of already installed versions of scipy and numpy. But it looks like the configuration requirements are different when I execute in a virtual environment; now numpy >= 1.15 is required, whereas in the original one numpy >= 1.13.3 was required. 

```
Collecting numpy>=1.15 (from seaborn->precice-config-visualizer==0.1.0)
  Downloading https://files.pythonhosted.org/packages/b8/e5/a64ef44a85397ba3c377f6be9c02f3cb3e18023f8c89850dd319e7945521/numpy-1.19.2-cp36-cp36m-manylinux1_x86_64.whl (13.4MB)

Collecting scipy>=1.0 (from seaborn->precice-config-visualizer==0.1.0)
  Using cached https://files.pythonhosted.org/packages/2b/a8/f4c66eb529bb252d50e83dbf2909c6502e2f857550f22571ed8556f62d95/scipy-1.5.2-cp36-cp36m-manylinux1_x86_64.whl
```

**Comment by fsimonis** [comment]
There is a simple solution to this problem.
I will drop the seaborn dependency and use a hard-coded colour palette instead. This should remove a significant amount of dependencies and prevent version conflicts.

**Comment by fsimonis** [comment]
This issue should be fixed by e3723d8e122ed77f16f0e9209aa5e65aca32532c.
Can you please check if this solves the problem on your setup?

**Comment by fsimonis** [comment]
Closing this issue as the dependency has been removed.
